psc-package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "purescript-prettier-printer",
+  "set": "psc-0.12.0",
+  "source": "https://github.com/purescript/package-sets.git",
+  "depends": [
+    "arrays",
+    "console",
+    "lists",
+    "prelude",
+    "psci-support",
+    "quickcheck",
+    "spec",
+    "strings",
+    "tuples"
+  ]
+}

```

Comment:
I don't know anything about `psc-package`.

@joneshf since you did something related to that in #9, does this look good to you?

Reply:
Seems fine to me, but I think the proper way to answer that question is to run it on CI. It's easy for this sort of duplication to get out of sync if it's not checked all the time.